### PR TITLE
Feature/individual page improvement

### PR DESCRIPTION
**Pull request by cvkumar**
Add Progression view and reduxify individual page view.

### CODE REVIEW

File: src/components/individuals/autocomplete.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ const styles = theme => ({
     root: {
         flexGrow: 1,
         height: 250,
-        // display: 'inline-block',
```
Comment (dannysperry) [inline src/components/individuals/autocomplete.js]: unneeded comment?

File: src/components/individuals/autocomplete.js [hunk 2]
```diff
@@ -111,9 +110,11 @@ class AutoComplete extends React.Component {
     }
 
     handleChange = (item) => {
-        // console.log(item)
-        if(item !== null) {
-            this.props.onSelect(item.beacon_holder.id)
+        console.log('handleChange()')
```
Comment (dannysperry) [inline src/components/individuals/autocomplete.js]: does this need to be here?

Comment (cvkumar) [reply src/components/individuals/autocomplete.js]: Good catch.

File: src/components/individuals/autocomplete.js (hunk 1 above)
Comment (cvkumar) [reply src/components/individuals/autocomplete.js]: Will remove!

**Pull request merged**